src/Workleap.OpenApi.MSBuild/GenerateContractProcess.cs

```diff
@@ -73,5 +91,23 @@ internal enum GenerateContractMode
 
         await Task.WhenAll(installationTasks);
         this._loggerWrapper.LogMessage("Finished installing OpenAPI dependencies.", MessageImportance.High);
+
+        var spectralRulesetPath = await spectralRulesetTask;
```

Comment:
Yes otherwise we have a warning. We could disable it but I think it is cleaner to leave the await and also is more defensive in case in the future someone remove this task from the Task.WhenAll list.